src/intrprtr.c

```diff
@@ -4659,9 +4659,9 @@ void             IntrAssertEnd3Args ( void )
 static Int InitKernel (
     StructInitInfo *    module )
 {
-    InitGlobalBag( &IntrResult, "src/intrprtr.c:IntrResult" );
-    InitGlobalBag( &IntrState,  "src/intrprtr.c:IntrState"  );
-    InitGlobalBag( &StackObj,   "src/intrprtr.c:StackObj"   );
+    InitGlobalBag( &TLS(IntrResult), "src/intrprtr.c:IntrResult" );
+    InitGlobalBag( &TLS(IntrState),  "src/intrprtr.c:IntrState"  );
+    InitGlobalBag( &TLS(StackObj),   "src/intrprtr.c:StackObj"   );
     InitCopyGVar( "CurrentAssertionLevel", &CurrentAssertionLevel );
```

Comment:
Perhaos `TLS` is also something we should change? As this is not about "thread local storage" anymore; and even if it was: Many people won't know what TLS stands for to begin with. I guess we could address the last bit by clearly documenting the `TLS` macro somewhere?

If we were to rename this, how about `GLOBAL` ? Or, if we keep using the term `InterpreterState`, this could be `ISTATE` or `INTR_STATE`, or ... Hm